src/compute/tests/table_v2_materialize.rs

```diff
@@ -286,12 +287,12 @@ async fn test_table_v2_materialize() -> Result<()> {
         column_nonnull! { F64Array, [1.14] },
     ];
     let chunk = DataChunk::builder().columns(columns.clone()).build();
-    let delete_inner: Box<dyn BatchExecutor> =
+    let delete_inner: BoxedExecutor2 =
```

Comment:
```suggestion
    let delete_inner: BoxedExecutor =
```

Will you rename `BoxedExecutor2` in later pr?

Reply:
Yes, it may introduce large conflicts, so I plan to do it in some midnight when dark moon and high wind.